File: src/URI/URI_1079.java
package URI;
import java.util.Locale;
import java.util.Scanner;

public class URI_1079 {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int n = sc.nextInt();
		
		for(int i = 0; i < n; i ++) {
			
			double nota1 = sc.nextDouble();
			double nota2 = sc.nextDouble();
			double nota3 = sc.nextDouble();
			double media = (nota1 * 2.0 + nota2 * 3.0 + nota3 * 5.0) / 10.0;
			System.out.printf("%.1f%n", media);
		}
		
		
		sc.close();

	}

}

File: kotlin/URI_1079.kt
import java.util.*

fun main(args: Array<String>) {
    Locale.setDefault(Locale.US)
    val sc = Scanner(System.`in`)
    val n = sc.nextInt()
    val calculaMedia = {a:Double, b: Double, c: Double ->
       (a * 2 + b * 3 + c * 5) / 10
    }
    repeat(n) {
        val a = sc.nextDouble()
        val b = sc.nextDouble()
        val c = sc.nextDouble()
        println("%.1f".format(calculaMedia(a,b,c)))
    }
}
